e2eTest/src/main/java/com/google/android/ground/e2etest/SurveyRunnerTest.kt

```diff
@@ -260,12 +259,14 @@ class SurveyRunnerTest : AutomatorRunner {
   }
 
   private fun setLoiName() {
-      takeScreenshot("${javaClass.simpleName}_${nameRule.methodName}")
+    takeScreenshot(getScreenshotName())
     if (device.wait(Until.hasObject(byText(R.string.save)), SHORT_TIMEOUT) == null) {
-      takeScreenshot("${javaClass.simpleName}_${nameRule.methodName}")
+      takeScreenshot(getScreenshotName())
       fail("Failed to find loi name popup")
     }
     enterText("An loi name")
     waitClickGone(byText(R.string.save))
   }
+
+  private fun getScreenshotName() = "${javaClass.simpleName}_${nameRule.methodName}"
```

Comment:
I was going to suggest this but I didn't realize this actually gets the test method name. Cool!

Reply:
Stupid question - if takeScreenshot() always has the same screenshot name logic, we can calculate the name there. The noargs takeScreenshot() would collide with the imported one; prob a good idea to rename in any case to avoid confusion between the helper and the actual imported API method.

Reply:
That's true... Done.